client/lib/abtest/active-tests.js

```diff
@@ -132,6 +132,14 @@ export default {
 		},
 		defaultVariation: 'keep',
 	},
+	gSuitePlan: {
+		datestamp: '20190117',
+		variations: {
+			basic: 95,
```

Comment:
Let's set `basic: 100` so we can separately address other parts of the UI with other PRs.